main.py

```diff
@@ -235,6 +234,11 @@ async def callback_handlers(bot: Client, cb: CallbackQuery):
                         c_time
                     )
                 )
+                if file_dl_path.rsplit('.', 1)[1].lower() != "mp4":
+                    os.system(f"ffmpeg -i {file_dl_path} -c copy {file_dl_path.rsplit('.', 1)[0]}.mp4")
+                    file_dl_path = file_dl_path.rsplit('.', 1)[0] + ".mp4"
+                os.system(f"ffmpeg -i {file_dl_path} -c:v copy -bsf:v h264_mp4toannexb -c:a aac {file_dl_path.rsplit('.', 1)[0]}.ts")
```

Comment:
Why need this? 👀
Please explain, because I don't know much but it. 😅

Reply:
Since I am noob and this filter(h264_mp4toannexb) is only for mp4s, I do not know any other way for non-mp4 videos.